### ISSUE TITLE
Color naming convention

### ISSUE THREAD

**Issue by MaggieCabrera** [WordPress/community-themes#193]
Let's come up with a naming convention and document it on the readme file so contributors can keep it consistent all over the different themes

**Comment by larakibby** [comment]
Color naming conventions can be fascinating, especially when it comes to digital media. Just like precise color codes help designers, having the right tool for converting YouTube videos to MP3 ensures a smooth experience. If you need a reliable converter, check out [YT to MP3](https://ytmp3.ing/)

**Comment by danish-036** [comment]
Yes you re right have some discussions on the [ invisible text](https://invisibletext.net/)
